cmd/generate.go

```diff
@@ -19,9 +20,10 @@ var language string
 const (
 	Go    = "go"
 	React = "react"
+	Kubernetes = "kubernetes"
 )
 
-var supportedLanguages = [...]string{Go, React}
+var supportedLanguages = [...]string{Go, React, Kubernetes}
```

Comment:
Maybe this should be called something else rather than "languages" if it's the right place for this. Or maybe we should start splitting this into sections.

Reply:
I agree, rename it modules? @Pritesh-Patel

Reply:
Don't worry about this for now, I'll do some refactoring.